contracts/protocol/clearinghouse/ClearingHouse.sol

```diff
@@ -50,6 +50,7 @@ contract ClearingHouse is
 
     error NotRageTradeFactory();
     error ZeroAmount();
+    error InvalidPoolId(uint32 poolId);
```

Comment:
We need to move this to IClearingHouseErrors in some PR.

Reply:
c8f6bb9257fa37858f8c629e2ff7baad8253e88e